plugin-api.d.ts

```diff
@@ -454,8 +458,11 @@ declare type StyleChangeProperty =
   | 'textDecoration'
   | 'letterSpacing'
   | 'lineHeight'
+  | 'leadingTrim'
   | 'paragraphIndent'
   | 'paragraphSpacing'
```

Comment:
listSpacing is a style property too I think (it should appear basically everywhere paragraph spacing does) (thank you!!)